src/RoleObjects/TabData.js

```diff
@@ -75,10 +76,13 @@ export default class TabData extends AccessibilityObject {
     return this._reactProps['aria-selected'];
   }
 
+
   /**
-   * Keydown listener for when the radio button is pressed
-   * @access private
-   */
+  * Keydown listener for when the radio button is pressed
+  * @access private
+  */
+
+
```

Comment:
Nit: please remove these blank lines.  A function comment should be right up against its associated function.

Reply:
Done